CONTRIBUTING.md

```diff
@@ -46,8 +46,24 @@ Finally, if you really, really, _really_ love developing PyBaMM, have a look at
 
 ## Installation
 
-To install PyBaMM with all developer options, type:
+To install PyBaMM with all developer options you can follow any one of the two steps described below:
 
+### 1. Using tox
+1. Install virtual environment using the command below:
+```bash
+pip install virtualenv
```

Comment:
I think Tox has `virtualenv` among its dependencies, so installing `tox` should automatically install `virtualenv`

Reply:
I'll remove that step.